src/ApiPort.VisualStudio/Utils/TargetPlatformDisplayNameConverter.cs

```diff
@@ -20,7 +20,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
             if (target.AlternativeNames.Count > 0)
             {
-                return $"{target.Name} ({string.Join(", ", target.AlternativeNames)})";
+                return FormattableString.Invariant($"{target.Name} ({string.Join(", ", target.AlternativeNames)})");
```

Comment:
Can you do that for an abstract class? `public abstract class FormattableString`.